packages/unified-latex-util-parse/libs/parse.ts

```diff
@@ -1,12 +1,28 @@
 import * as Ast from "@unified-latex/unified-latex-types";
-import { unified } from "unified";
+import { type FrozenProcessor, unified } from "unified";
 import { unifiedLatexFromString } from "./plugin-from-string";
+import type { PluginOptions } from "./plugin-from-string";
 
-const parser = unified().use(unifiedLatexFromString).freeze();
+let parser = unified().use(unifiedLatexFromString).freeze();
```

Comment:
Was there a reason this needed to be changed to `let`?

Reply:
Indeed, no. That change was made on a previous attempt, which was later reverted. I think const would be perfectly fine here.